### PR TITLE
Refactor entire app to use css modules

### PR DESCRIPTION
**Pull request by this-Cunningham**

### CODE REVIEW

**Comment by this-Cunningham** [comment]
[CSS Modules refactor](https://trello.com/c/CLE82M7B/66-css-modules-refactor)

File: spec/components/Overview.react.test.jsx [hunk 1]
```diff
@@ -12,6 +12,9 @@ describe('<Overview />', () => {
     // app = shallow(<Attraction />);
     wrapper = mount(<Overview overview={data.overview} />);
   });
+  afterEach(() => {
+    wrapper.unmount();
+  });
```
Comment (sabateklu) [inline spec/components/Overview.react.test.jsx]: DRY code- way to go

File: spec/components/Images.react.test.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    images.setProps({ images: [] });
+    expect(images.find('button')).toHaveLength(0);
+  });
+  test('Images should change to next/prev image if next/prev button is clicked ', () => {
+    const instance = images.instance();
+    const spy = jest.spyOn(instance, 'changeImage');
+    const nextButton = images.find('button.next');
+    const prevButton = images.find('button.prev');
+    expect(instance.state.displayedImage).toEqual('https://d3tteoxp56tq1d.cloudfront.net/images/tripadvisor_thailand_10.jpg');
+    nextButton.simulate('click');
```
Comment (sabateklu) [inline spec/components/Images.react.test.jsx]: great use of dynamic testing methods.

File: src/components/Images.jsx [hunk 3]
```diff
@@ -6,20 +6,63 @@ export default class Images extends React.Component {
...
+  componentDidMount() {
     const { images } = this.props;
+    const sliced = images.slice();
+    this.setState({
+      attractionImages: sliced,
+      displayedImage: sliced[0],
+    });
+  }
+
+  changeImage(direction) {
```
Comment (sabateklu) [inline src/components/Images.jsx]: your implementation of the prev and next buttons here to change state are clear and concise- good job!

**Review by sabateklu — APPROVED** [review]
looks like the css modules implementation wasn't too bad. great job :)

**Pull request merged**